apps/ledger-live-desktop/src/renderer/screens/settings/sections/Experimental/PostOnboardingHubTester.tsx

```diff
@@ -20,7 +20,13 @@ const PostOnboardingHubTester = () => {
     >
       <Button
         data-test-id="postonboarding-tester-button"
-        onClick={() => handleInitStax(DeviceModelId.stax, true, () => history.push("/"))}
+        onClick={() =>
+          handleInitPostOnboarding({
+            deviceModelId: DeviceModelId.stax,
+            mock: true,
+            fallbackIfNoAction: () => history.push("/"),
+          })
```

Comment:
- this function's signature changed, now passing options in an object for more clarity